keyboards/coseyfannitutti/discipline/rules.mk

```
@@ -36,6 +36,4 @@ UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
-HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
-
-LAYOUTS = 65_ansi
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
```

Any reason community support was removed? Given you are adding iso support...


```suggestion
HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs

LAYOUTS = 65_ansi 65_iso
```

---

65_ansi was causing travis to fail for some reason

---

looks fine now, though.

---

Its fine if a community layout fails to build for a reason like "user assumed board has audio capability", but not fine if its due to LAYOUT macro incompatibilities.

Removing it will remove a bunch of keymaps from being compiled, which i guess it why it builds now. However this really should go back if it can.

---

Yeah, 65_ansi is failing.

---

It's passing too many arguments.

---

Thats fine, its just an issue with merlins keymap, the rest compiled fine.

---

I agree with zvecr here. 

At worst, @mechmerlin should address it in his keymap.